eodhistoricaldata/src/endpoint/price.ts

```diff
@@ -0,0 +1,45 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, Config } from '@chainlink/types'
+import { util } from '@chainlink/ea-bootstrap'
+
+export const Name = 'price'
+
+const customError = (data: any) => data.Response === 'Error'
+
+const customParams = {
+  base: ['base', 'asset', 'from', 'symbol'],
+  endpoint: false,
+}
+
+const commonKeys: { [key: string]: string } = {
+  N225: 'N225.INDX',
+  FTSE: 'FTSE.INDX',
+  BZ: 'BZ.COMM',
+}
+
+export const execute = async (config: Config, request: AdapterRequest) => {
```

Comment:
```suggestion
export const execute: ExecuteWithConfig<Config> = async (request, config) => {
```
A recent PR made ExecuteWithConfig a generic. (This applies across all EAs in this PR)